salt/modules/zabbix.py

```diff
@@ -2503,3 +2503,187 @@ def configuration_import(config_file, rules=None, file_format='xml', **connectio
                                                                 'called successfully.'}
     except SaltException as exc:
         return {'name': config_file, 'result': False, 'message': six.text_type(exc)}
+
+
+def triggerid_get(hostid=None, trigger_desc=None, priority=4, **connection_args):
```

Comment:
We typically prefer named kwargs instead of `**connection_args`)

Reply:
good idea. I changed it in whole module.